src/utils/perfherder/chartJsFormatter.js

```diff
@@ -0,0 +1,60 @@
+const dataToChartJSformat = data =>
+  data.map(({ datetime, value }) => ({
+    x: datetime,
+    y: value,
+  }));
+
+const chartJsOptions = (reverse, scaleLabel) => ({
+  scales: {
+    xAxes: [
+      {
+        type: 'time',
+        time: {
+          displayFormats: {
+            hour: 'MMM D',
+          },
+        },
+      },
+    ],
+    yAxes: [
+      {
+        ticks: {
+          beginAtZero: true,
+          reverse,
+        },
+        scaleLabel: {
+          display: true,
+          labelString: scaleLabel,
+        },
+      },
+    ],
+  },
+});
+
+const generateChartJsOptions = (meta) => {
+  const higherIsBetter = (meta.lower_is_better === false);
+  const reversed = higherIsBetter;
+  const yLabel = higherIsBetter ? 'Score' : 'Execution time (ms)';
+  return chartJsOptions(reversed, yLabel);
+};
+
+// This function combines Perfherder series and transforms it into ChartJS formatting
+const chartJsFormatter = (series) => {
+  const newData = {
+    data: { datasets: [] },
+    // The first series defines the whole set
+    options: generateChartJsOptions(series[0].meta),
+  };
+
+  series.forEach(({ color, data, label }) => {
+    newData.data.datasets.push({
+      backgroundColor: color,
+      label: label,
```

Comment:
nit: you could use the ES6 property value shorthand with `label` instead of `label: label`.